### ISSUE TITLE
Krecognition: Complete screens

### ISSUE THREAD

**Issue by paddyow** [Kademi/kademi-public#159]
When all levels are achieved, the current 'end screen' looks crap. Suggest the following:

The horizontal progress bar
- Add an option to show the complete message which where the label can be edited.  Sometimes when there are no new levels to achieved, we might not want to show it.
- Update the current period progress panels to show a complete message and more contextual content which will vary based on a few factors

![image](https://user-images.githubusercontent.com/2201340/115508575-cfe14680-a2c0-11eb-80ea-4d2eeaf30291.png)

**Comment by bradmac** [comment]
> Update the current period progress panels to show a complete message and more contextual content which will vary based on a few factors

I dont think this relates specifically to not having further levels. Users should always be able to see that regardless of whether there are more levels to achieve or not
